### PR TITLE
[LIBS-347] add publish-languages command

### PR DESCRIPTION
**Pull request by GZLiew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,31 @@ A Buildkite library plugin with common utilities used in CI.
...
+```yml
+steps:
+  - command: publish-languages
+    env:
+      # optional defaults to frontend/locale/en.json
+      PUBLISH_LANGUAGES_FILE_PATH: en.json
+      PUBLISH_FEATURES_EVENT_BUS_NAME: busname
+      # service_name is inferred from repo URL
+      PUBLISH_FEATURES_EVENT_USER_ID: userid
+      PUBLISH_FEATURES_DETAIL_TYPE: languages.updated
```
Comment (smilin-desperado) [inline README.md:27]: **question:** Should these be `PUBLISH_LANGUAGES`?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,31 @@ A Buildkite library plugin with common utilities used in CI.
...
 
+### publish-languages
+
+Publish languages.json file to EventBridge
+
+```yml
+steps:
+  - command: publish-languages event_bus_name service_name user_id event_detail_type
+    plugins:
+      - tapendium/tap-ci-utils#v0.4.1: ~
```
Comment (smilin-desperado) [inline README.md:15]: **nit:** Should this be minor version bump rather than a patch because it's new functionality?

Comment (GZLiew) [reply README.md]: good point! 583c90b

**Review by smilin-desperado — APPROVED** [review]
🚀

**Pull request merged**